src/elements/content-open-with/ContentOpenWith.js

```diff
@@ -192,7 +192,7 @@ class ContentOpenWith extends PureComponent<Props, State> {
         }
 
         if (currentFileId !== previousFileId) {
-            this.setState({ isLoading: true, integrations: null });
+            this.setState({ ...this.initialState });
```

Comment:
https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops

> If you want to “reset” some state when a prop changes, consider either making a component fully controlled or fully uncontrolled with a key instead.

Reply:
Good idea, since I'm already double rendering I'll merge this fix in for now and work on a permanent solution next week.